### DISCUSSION TITLE
Move to Required Workflows

### DISCUSSION THREAD

**Discussion by smorimoto** [asdf-community/.github#49] [General]
GitHub recently announced required workflows and I feel like it really fits our organisation.

https://github.blog/2023-01-10-introducing-required-workflows-and-configuration-variables-to-github-actions/

I think some people are starting to realise that as more plugins are being maintained in our organization, the fact that "repository per plugin" is causing huge maintenance costs.
So, let me just share that we are planning to move some workflows to this in the next few weeks. If you have any concerns, let me know on this thread.

**Comment by smorimoto** [comment]
This workflow no longer runs if the Makefile is not found in the repository and prints a warning instead.

**Comment by smorimoto** [comment]
If the GitHub Actions workflow file exists in the repository now it checks linting with actionlint.

**Comment by barraponto** [comment]
I was amazed by it when I learned about it (yesterday).
Then I dug into documentation and found out it was [deprecated in favor of Repository Rules](https://github.blog/changelog/2023-08-02-github-actions-required-workflows-will-move-to-repository-rules/).
Are we moving on to repository rules?
